csrc/posix/pf_io_posix.c

```diff
@@ -131,6 +131,7 @@ void sdTerminalInit(void)
         {
             perror("sdTerminalInit: tcsetattr");
         }
+    setvbuf(stdout, NULL, _IONBF, (size_t) 0);
```

Comment:
setvbuf should be indented

Reply:
Ok, indented, and now we check for error.

Reply:
Also... `setvbuf` is standard in C99, and its prototype is in `stdio.h`.

For reference, I see it's mentioned in Harbison & Steele "C: a reference manual", 5th edition, section 15.3 (page 370).